test/e2e/cluster/stack.go

```diff
@@ -121,6 +121,10 @@ func (s *stack) deploy(ctx context.Context, test TestResources) (*resourcesStack
 			ParameterKey:   aws.String("RolePathPrefix"),
 			ParameterValue: aws.String(constants.TestRolePathPrefix),
 		},
+		{
+			ParameterKey:   aws.String("CreationTime"),
+			ParameterValue: aws.String(time.Now().Format(time.RFC3339)),
```

Comment:
this probably updates the tag value every time you the `setup` command, right? which means all resources with the tag get updated